source/server/connection_handler_impl.h

```diff
@@ -109,6 +109,31 @@ class ConnectionHandlerImpl : public Network::ConnectionHandler, NonCopyable {
 
   typedef std::unique_ptr<ActiveListener> ActiveListenerPtr;
 
+  // TODO(conqerAtapple): Add implementation.
```

Comment:
I think I need to see more of what we plan to do here before I can give a super useful review, sorry :-/